typescript/test/utils/readTestData.ts

```diff
@@ -242,6 +253,26 @@ function mapPool(
             priceRatioUpdateEndTime: BigInt(pool.priceRatioUpdateEndTime),
         };
     }
+    if (pool.poolType === 'LIQUIDITY_BOOTSTRAPPING') {
+        return {
+            ...pool,
+            scalingFactors: pool.scalingFactors.map((sf) => BigInt(sf)),
+            swapFee: BigInt(pool.swapFee),
+            balancesLiveScaled18: pool.balancesLiveScaled18.map((b) =>
+                BigInt(b),
+            ),
+            tokenRates: pool.tokenRates.map((r) => BigInt(r)),
+            totalSupply: BigInt(pool.totalSupply),
+            weights: (
+                pool as TransformBigintToString<LiquidityBootstrappingPool>
+            ).weights.map((w) => BigInt(w)),
+            aggregateSwapFee: BigInt(pool.aggregateSwapFee ?? '0'),
+            supportsUnbalancedLiquidity:
```

Comment:
Can default to false with a comment added to explain why we're doing that.

Reply:
added your suggestion